sale_order_line_price_history/wizards/sale_order_line_price_history.py

```diff
@@ -113,7 +113,6 @@ class SaleOrderLinePriceHistoryline(models.TransientModel):
     )
     sale_order_line_id = fields.Many2one(
         comodel_name="sale.order.line",
-        string="Sale order line",
```

Comment:
Why removing this as this will generate 'Sale Order Line Id' as base string for translations?

Reply:
In fact it will just generate `'Sale Order Line'` (with upper), and I think it's because a pylint linter bug that didn't take into account upper and lower, saying that "it's redundant". It's fixed, but maybe the version with the fix is not yet here.

Reply:
I remove it because I had problems with target_sale_order_line_id label